scripts/cli.js

```diff
@@ -170,11 +167,11 @@ function cmdLoad(configPath, privateKeyPath) {
   });
 }
 
-function cmdConvert(privateKeyPath, backup) {
+function cmdConvert(privateKeyPath, backup, normalizeKey) {
   getPrivateKeyPath(privateKeyPath, function (privateKeyPath) {
     getBackup(backup, function (backup) {
       rl.write(': converting...');
-      instance.convert({ privateKeyPath: privateKeyPath, backup: backup === 'true' });
+      instance.convert({ privateKeyPath: privateKeyPath, backup: backup === 'true', normalizeKey: (normalizeKey === 'true' || instance.config._.normalizeKey) });

```

Comment:
To keep things simple I'd suggest making this default to enabled, and we can bump major version.